bootstrap.d/dev-util.yml

```diff
@@ -53,6 +53,25 @@ tools:
     install:
       - args: ['make', 'install']
 
+  - name: host-tzdb-2024a
+    architecture: '@OPTION:arch@'
+    source:
+      subdir: 'ports'
+      url: 'https://data.iana.org/time-zones/releases/tzdb-2024a.tar.lz'
+      format: 'tar.xz'
+      checksum: blake2b:91d6c55c4f860ced760df84a1751174f1eb8ea83939b44d3a206dccbc9bac1713fbcde70391deaa4ed01d95e6de08bc0b751d296bebdf5d9ca0b4534942694cf
+      extract_path: 'tzdb-2024a'
+      version: '2024a'
+    tools_required:
+      - system-gcc
```

Comment:
Considering it's a tool, do we really need the compiler that targets managarm here?

Reply:
I wasnt sure, but removing it didnt break it. fixed